packages/esm-patient-list-app/src/index.ts

```diff
@@ -31,15 +31,15 @@ function setupOpenMRS() {
     },
     {
       path: `${spaBasePath}/:view?`,
-      title: ([x]) => `${x} Dashboard`,
+      title: ([x]) => 'Dashboard',
```

Comment:
Is this accurate or should it be "Members"? I guess the `:view?` may also be `:view` (i.e., non-optional, this has to be given)